tasks.rb

```diff
@@ -2406,6 +2407,11 @@ def set_measure_argument_values(hpxml_file, args, sch_args)
     sch_args['schedules_type'] = 'smooth'
     sch_args['output_csv_path'] = '../../HPXMLtoOpenStudio/resources/schedule_files/smooth.csv'
     sch_args['hpxml_output_path'] = sch_args['hpxml_path']
+  elsif ['base-schedules-detailed-stochastic-10-mins.xml'].include? hpxml_file
+    sch_args['hpxml_path'] = args['hpxml_path']
+    sch_args['schedules_type'] = 'stochastic'
+    sch_args['output_csv_path'] = '../../HPXMLtoOpenStudio/resources/schedule_files/stochastic-10-mins.csv'
+    sch_args['hpxml_output_path'] = sch_args['hpxml_path']
```

Comment:
Seems like a good addition. What prompted you to add it?

Reply:
After having talked with @yzhou601 about generating minutely schedules, I noticed we had no workflow tests for subhourly schedules.